Formula/sleepwatcher.rb

```diff
@@ -18,12 +18,12 @@ def install
     # Adjust Makefile to build native binary only
     inreplace "sources/Makefile" do |s|
       s.gsub! /^(CFLAGS)_PPC.*$/, "\\1 = #{ENV.cflags} -prebind"
-      s.gsub! /^(CFLAGS_X86)/, "#\\1"
+      s.gsub! /^(CFLAGS_I386|CFLAGS_X86_64)/, "#\\1"
```

Comment:
What are we hacking here exactly?

Reply:
From v2.2 to v2.2.1 the makefile changed. Now not only the line starting with `CFLAGS_X86` has to be commented out, but also the line starting with `CFLAGS_X86_64`. See below for ‘before’ and ‘after’ for both versions.

### _Old version 2.2_ makefile source …:
```#
#	Makefile for sleepwatcher
#
#	21.11.2004 bb	created
#	05.05.2005 bb	removed Carbon framework, target clean added
#	23.04.2006 bb	produce a Universal binary of sleepwatcher
#	02.04.2010 bb	-macosx-version-min=10.3 => 10.4
#

CFLAGS_PPC= -O3 -prebind -mmacosx-version-min=10.4 -mtune=G4 -arch ppc
CFLAGS_X86= -O3 -prebind -mmacosx-version-min=10.4 -mtune=pentium -arch i386
LIBS= -framework IOKit -framework CoreFoundation

BINDIR=/usr/local/sbin
MANDIR=/usr/local/man

sleepwatcher: sleepwatcher.c
	$(CC) $(CFLAGS_PPC) -o sleepwatcher.ppc sleepwatcher.c $(LIBS)
	$(CC) $(CFLAGS_X86) -o sleepwatcher.x86 sleepwatcher.c $(LIBS)
	lipo -create sleepwatcher.ppc sleepwatcher.x86 -output sleepwatcher
	rm -f sleepwatcher.ppc sleepwatcher.x86

install: sleepwatcher sleepwatcher.8
	mkdir -p $(BINDIR)
	install -o root -g wheel -m 755 sleepwatcher $(BINDIR)
	mkdir -p $(MANDIR)/man8
	install -o root -g wheel -m 644 sleepwatcher.8 $(MANDIR)/man8

clean:
	rm -f sleepwatcher
```
**… and patched:**
```#
#	Makefile for sleepwatcher
#
#	21.11.2004 bb	created
#	05.05.2005 bb	removed Carbon framework, target clean added
#	23.04.2006 bb	produce a Universal binary of sleepwatcher
#	02.04.2010 bb	-macosx-version-min=10.3 => 10.4
#

CFLAGS =  -prebind
#CFLAGS_X86= -O3 -prebind -mmacosx-version-min=10.4 -mtune=pentium -arch i386
LIBS= -framework IOKit -framework CoreFoundation

BINDIR=$(PREFIX)/sbin
MANDIR=$(PREFIX)/share/man

sleepwatcher: sleepwatcher.c
	$(CC) $(CFLAGS) -o sleepwatcher sleepwatcher.c $(LIBS)
#	$(CC) $(CFLAGS_X86) -o sleepwatcher.x86 sleepwatcher.c $(LIBS)
#	lipo -create sleepwatcher.ppc sleepwatcher.x86 -output sleepwatcher
#	rm -f sleepwatcher.ppc sleepwatcher.x86

install: sleepwatcher sleepwatcher.8
	mkdir -p $(BINDIR)
	install  -m 755 sleepwatcher $(BINDIR)
	mkdir -p $(MANDIR)/man8
	install  -m 644 sleepwatcher.8 $(MANDIR)/man8

clean:
	rm -f sleepwatcher
```

### _New version 2.2.1_ makefile source …:
```#
#	Makefile for sleepwatcher
#
#	21.11.2004 bb	created
#	05.05.2005 bb	removed Carbon framework, target clean added
#	23.04.2006 bb	produce a Universal binary of sleepwatcher
#	02.04.2010 bb	-macosx-version-min=10.3 => 10.4
#	26.12.2018 bb	modified for 64-bit build, PowerPC now optional
#			use "make sleepwatcher" to build the Intel only binary,
#			use "make sleepwatcher.ppc" in a Xcode 3.2.6
#			environment to create the PowerPC binary, then
#			use "make fat" to create a fat binary (PowerPC and
#			Intel 32-bit and 64-bit)
#

CFLAGS_PPC= -O3 -prebind -mmacosx-version-min=10.4 -mtune=G4 -arch ppc
CFLAGS_I386= -O3 -prebind -mmacosx-version-min=10.4 -arch i386
CFLAGS_X86_64= -O3 -prebind -mmacosx-version-min=10.4 -arch x86_64 
LIBS= -framework IOKit -framework CoreFoundation

BINDIR=/usr/local/sbin
MANDIR=/usr/local/man

sleepwatcher: sleepwatcher.c
	$(CC) $(CFLAGS_I386) -o sleepwatcher.i386 sleepwatcher.c $(LIBS)
	$(CC) $(CFLAGS_X86_64) -o sleepwatcher.x86_64 sleepwatcher.c $(LIBS)
	lipo -create sleepwatcher.i386 sleepwatcher.x86_64 -output sleepwatcher
	rm sleepwatcher.i386 sleepwatcher.x86_64

fat: sleepwatcher sleepwatcher.ppc
	lipo -create sleepwatcher sleepwatcher.ppc -output sleepwatcher.fat
	mv sleepwatcher.fat sleepwatcher
	rm sleepwatcher.ppc

sleepwatcher.ppc: sleepwatcher.c
	$(CC) $(CFLAGS_PPC) -o sleepwatcher.ppc sleepwatcher.c $(LIBS)

install: sleepwatcher sleepwatcher.8
	mkdir -p $(BINDIR)
	install -o root -g wheel -m 755 sleepwatcher $(BINDIR)
	mkdir -p $(MANDIR)/man8
	install -o root -g wheel -m 644 sleepwatcher.8 $(MANDIR)/man8

clean:
	rm -f sleepwatcher
```
**… and patched:**
```#
#	Makefile for sleepwatcher
#
#	21.11.2004 bb	created
#	05.05.2005 bb	removed Carbon framework, target clean added
#	23.04.2006 bb	produce a Universal binary of sleepwatcher
#	02.04.2010 bb	-macosx-version-min=10.3 => 10.4
#	26.12.2018 bb	modified for 64-bit build, PowerPC now optional
#			use "make sleepwatcher" to build the Intel only binary,
#			use "make sleepwatcher.ppc" in a Xcode 3.2.6
#			environment to create the PowerPC binary, then
#			use "make fat" to create a fat binary (PowerPC and
#			Intel 32-bit and 64-bit)
#

CFLAGS =  -prebind
#CFLAGS_I386= -O3 -prebind -mmacosx-version-min=10.4 -arch i386
#CFLAGS_X86_64= -O3 -prebind -mmacosx-version-min=10.4 -arch x86_64 
LIBS= -framework IOKit -framework CoreFoundation

BINDIR=$(PREFIX)/sbin
MANDIR=$(PREFIX)/share/man

sleepwatcher: sleepwatcher.c
	$(CC) $(CFLAGS) -o sleepwatcher sleepwatcher.c $(LIBS)
#	$(CC) $(CFLAGS_X86_64) -o sleepwatcher.x86_64 sleepwatcher.c $(LIBS)
#	lipo -create sleepwatcher.i386 sleepwatcher.x86_64 -output sleepwatcher
#	rm sleepwatcher.i386 sleepwatcher.x86_64

fat: sleepwatcher sleepwatcher.ppc
	lipo -create sleepwatcher sleepwatcher.ppc -output sleepwatcher.fat
	mv sleepwatcher.fat sleepwatcher
	rm sleepwatcher.ppc

sleepwatcher.ppc: sleepwatcher.c
	$(CC) $(CFLAGS_PPC) -o sleepwatcher.ppc sleepwatcher.c $(LIBS)

install: sleepwatcher sleepwatcher.8
	mkdir -p $(BINDIR)
	install  -m 755 sleepwatcher $(BINDIR)
	mkdir -p $(MANDIR)/man8
	install  -m 644 sleepwatcher.8 $(MANDIR)/man8

clean:
	rm -f sleepwatcher
```